### PR TITLE
Changes order of ETD display metadata

### PR DESCRIPTION
**Pull request by rotated8**
Moves the degree, department, and school up under the creator.

### CODE REVIEW

**Comment by rotated8** [comment]
Resolves #2

File: spec/features/show_etd_spec.rb [hunk 1]
```diff
@@ -34,6 +34,11 @@
       expect(page).to have_content etd.degree.first
       expect(page).to have_content etd.department.first
       expect(page).to have_content etd.school.first
+
+      headers = page.all(:xpath, './/table[contains(@class, "attributes")]/tbody/tr/th')
+      expect(headers[0].text).to eq 'Creator'
+      expect(headers[1].text).to eq 'Degree'
+      expect(headers[2].text).to eq 'Department'
```
Comment (Steve-A-Orr) [inline spec/features/show_etd_spec.rb]: Did you intentionally mean to leave out **School**?

Comment (Steve-A-Orr) [inline spec/features/show_etd_spec.rb]: Looks fine.

Comment (rotated8) [inline spec/features/show_etd_spec.rb]: :wink:

**Pull request merged**